app/views/journeys/expenses/travelAndAccommodation/VehicleExpensesView.scala.html

```diff
@@ -0,0 +1,93 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.journeys.expenses.travelAndAccommodation.routes
+@import viewmodels.InputWidth._
+@import viewmodels.LabelSize
+@import models.common.{BusinessId, TaxYear, UserType}
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+@import views.html.components._
+@import models.journeys.expenses.travelAndAccommodation.TravelAndAccommodationExpenseType
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.label.Label
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukInput: GovukInput,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode, userType: UserType, taxYear: TaxYear, businessId: BusinessId, expenseType: Set[TravelAndAccommodationExpenseType])(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages(s"vehicleExpenses.title.$userType"))) {
+
+    @formHelper(action = routes.VehicleExpensesController.onSubmit(taxYear, businessId, mode)) {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value")))
+        }
+
+        <h1 class="govuk-heading-l">@messages(s"vehicleExpenses.title.$userType")</h1>
```

Comment:
We can use the componenet  @heading()
```suggestion
 @heading(s"vehicleExpenses.title.$userType"`)
      
```

Reply:
amended to @Heading